common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1180,17 +1192,45 @@ object RssConf extends Logging {
     }
   }
 
-  def HDDFlusherThread(conf: RssConf): Int = {
-    conf.getInt("rss.flusher.hdd.thread.count", 1)
-  }
+  val WORKER_FLUSHER_HDD_THREAD_COUNT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.hdd.thread.count")
+      .withAlternative("rss.flusher.hdd.thread.count")
+      .categories("worker")
+      .doc("Flusher's thread count used for write data to HDD disks.")
+      .intConf
+      .createWithDefault(1)
 
-  def SSDFlusherThread(conf: RssConf): Int = {
-    conf.getInt("rss.flusher.ssd.thread.count", 8)
-  }
+  val WORKER_FLUSHER_SSD_THREAD_COUNT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.ssd.thread.count")
+      .withAlternative("rss.flusher.hdd.thread.count")
+      .categories("worker")
+      .doc("Flusher's thread count used for write data to SSD disks.")
+      .intConf
+      .createWithDefault(8)
 
-  def diskMinimumReserveSize(conf: RssConf): Long = {
-    Utils.byteStringAsBytes(conf.get("rss.disk.minimum.reserve.size", "5G"))
-  }
+  val WORKER_FLUSHER_HDFS_THREAD_COUNT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.hdfs.thread.count")
+      .withAlternative("rss.worker.hdfs.flusher.thread.count")
+      .categories("worker")
+      .doc("Flusher's thread count used for write data to HDFS.")
+      .intConf
+      .createWithDefault(4)
+
+  val WORKER_DISK_RESERVATION: ConfigEntry[Long] =
+    buildConf("celeborn.worker.disk.reservation")
+      .withAlternative("rss.disk.minimum.reserve.size")
+      .categories("worker")
+      .doc("Celeborn worker reserved space for each disk.")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("5G")
+
+  def HDDFlusherThread(conf: RssConf): Int = conf.get(WORKER_FLUSHER_HDD_THREAD_COUNT)
```

Comment:
We can move such method to `RssConf` class then just call `conf. HDDFlusherThread()`

Reply:
Sounds good to me.